emukit/examples/dynamic_negative_lower_confidence_bound/dnlcb.py

```diff
@@ -0,0 +1,28 @@
+from emukit.bayesian_optimization.acquisitions import NegativeLowerConfidenceBound
+from emukit.core.interfaces import IModel, IDifferentiable
+import numpy as np
+from typing import Union
+
+class DynamicNegativeLowerConfidenceBound(NegativeLowerConfidenceBound):
+    def __init__(self, model: Union[IModel, IDifferentiable], input_space_size: int, delta: float) -> None:
+        """
```

Comment:
That's a very nice docstring here. Just one thing about it: please add reference to the paper.

Reply:
The parent class is referenced, explicitly. This is found in the very same paper and I thought a repeated reference is notf fine.